From 18c6dbd17de3750790eeba455770b807c57e9965 Mon Sep 17 00:00:00 2001
From: Yasufumi Ogawa <yasufum.o@gmail.com>
Date: Tue, 10 Sep 2019 20:06:25 +0900
Subject: [PATCH 1/2] spp_mirror: remove unit test section

Spp_mirror supports unit test which was used before released, but nouse
anymore. This update is to remove the feature.

Signed-off-by: Yasufumi Ogawa <yasufum.o@gmail.com>
---
 src/mirror/spp_mirror.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/src/mirror/spp_mirror.c b/src/mirror/spp_mirror.c
index 835c760..0ace09c 100644
--- a/src/mirror/spp_mirror.c
+++ b/src/mirror/spp_mirror.c
@@ -615,12 +615,7 @@ main(int argc, char *argv[])
 
 		/* Enter loop for accepting commands */
 		int ret_do = 0;
-#ifndef USE_UT_SPP_VF
-		while (likely(g_core_info[master_lcore].status !=
-				SPPWK_LCORE_REQ_STOP)) {
-#else
 		{
-#endif
 			/* Receive command */
 			ret_do = sppwk_run_cmd();
 			if (unlikely(ret_do != SPPWK_RET_OK))
-- 
2.17.1

